recyclerview/src/main/java/me/henrytao/recyclerview/BaseAdapter.java

```diff
@@ -126,10 +138,14 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
   }
 
   public int getBaseItemCount() {
-    if (mBaseAdapter == null || !mIsBaseAdapterEnabled) {
+    if (mBaseAdapters == null || !mIsBaseAdapterEnabled) {
       return 0;
     }
-    return mBaseAdapter.getItemCount();
+    int count = 0;
+    for (RecyclerView.Adapter adapter : mBaseAdapters) {
```

Comment:
unnecessary to define `int n = mBaseAdapters.length` because mBaseAdapters.length is in memory